editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -467,6 +467,7 @@ class CanvasItemEditor : public VBoxContainer {
 
 	void _draw_viewport();
 
+	void _is_hovering_guide(Point2 p_pos, bool p_is_pressed = false);
```

Comment:
This will be called twice, when a mouse motion or mouse pressed events happens, this will prevent duplicating code since both should check if we are hovering a guide to either change cursor shape or to start dragging.